haas/plugins/result_handler.py

```diff
@@ -217,3 +221,63 @@ def __call__(self, result):
             self.stream.write(" '{0}'".format(result.message))
         self.stream.writeln()
         self.stream.flush()
+
+
+class SlowTestsResultHandler(IResultHandlerPlugin):
+    separator1 = '=' * 70
+    separator2 = separator2
+
+    OPTION_DEFAULT = object()
```

Comment:
True, this always gets me (Apparently brain functions need food and coffee). Please ignore my comment